swift/HtmlConverter/Tests/HtmlConverterTests.swift

```diff
@@ -0,0 +1,26 @@
+import XCTest
+
+final class HtmlConverterTests: XCTestCase {
+    func test_convertToHtml_usesValidHtmlTags() {
+        // TODO: write this test
+    }
+
+    func test_convertToHtml_usesValidParagraphTags() {
+        // TODO: write this test
+    }
+}
+
+fileprivate class StubTwoLinesFileReader {
```

Comment:
Is it duck typing like Python? I expected this class to implement some kind of interface to indicate it is a kind of file-like object

Reply:
Oh! No, sorry if this is confusing. I saw this in the C# project and thought this was meant as a clue for the developer:
  "Both StubTwoLinesFileReader and EmptyLinesFileReader have a 'readLines' function, so maybe that's a kind of dependency to inject to the HtmlConverter ?"
  
In Swift we may define a protocol (i.e. an interface) that the types would conform to (I believe you can think of the protocol almost as an abstract class)
Then we would use this protocol to define the dependencies of a class

Reply:
These two test double classes are supposed to be ready-made replacements for the actual file so when you write the tests you can use them as a kind of in-memory file. It seems to me they won't work for that unless they implement the protocol of a file. So I think this might need fixing?

Reply:
Agreed 👍 I'll work on that as soon as I can

Reply:
Agreed 👍 I'll work on that as soon as I can